stl/src/_tolower.cpp

```diff
@@ -17,6 +17,8 @@
 #undef _tolower
 #undef tolower
 
+#define _cpp_isleadbyte(c) (__pctype_func()[static_cast<unsigned char>(c)] & _LEADBYTE)

```

Comment:
I'm not too happy about duplicating this definition in two source files. Could we move it into an internal header like `awint.hpp` or something instead?